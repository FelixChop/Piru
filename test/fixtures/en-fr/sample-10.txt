The artist's magnum opus was lauded for its visceral imagery, leaving critics at a loss for words.
