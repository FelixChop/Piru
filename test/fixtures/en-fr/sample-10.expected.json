[
  {
    "word": "magnum opus",
    "lemma": "magnum opus",
    "definition": "chef-d'œuvre",
    "citation": "The artist's magnum opus was lauded for its visceral imagery, leaving critics at a loss for words.",
    "to_learn": true
  },
  {
    "word": "lauded",
    "lemma": "to laud",
    "definition": "louer, faire l'éloge de quelque chose ou quelqu'un",
    "citation": "The artist's magnum opus was lauded for its visceral imagery, leaving critics at a loss for words.",
    "to_learn": true
  },
  {
    "word": "at a loss for words",
    "lemma": "at a loss for words",
    "definition": "sans voix",
    "citation": "The artist's magnum opus was lauded for its visceral imagery, leaving critics at a loss for words.",
    "to_learn": true
  }
]
